GVFS/GVFS.Upgrader/UpgradeOrchestrator.cs

```diff
@@ -219,6 +219,25 @@ private bool TryRunUpgrade(out Version newVersion, out string consoleError)
                 return false;
             }
 
+            if (!this.LaunchInsideSpinner(

```

Comment:
Mainly curious - what do we get by breaking this out into its own step now? Is it just more granular messaging about what the upgrader is doing? Should it be more generic `Checking for blocking processes`, or even `Performing pre-upgrade checks`? alternatively should we keep with the existing top level messages and minimize change for 147? I do think it was a bit overloaded that we (used to) perform this check as part of unmounting repositories, so I think this fits better.

Reply:
`Checking for blocking processes` makes more sense than you!  If you look at the title we were putting blocking processes as a failure for unmounting repos.  That wasn't actually an accurate description of what was happening.

Reply:
To elaborate previously we told the user "Unmounting failed" when that isn't what happened (see summary at top for example).  That is likely to be confusing to a user.  Now we let the user know unmounting was successful but they have git processes that are blocking.  Since I moved existing code I think it's a safe change we should take.

Reply:
Yeah - I think it makes more sense with your changes. Thanks!